src/OrbitGl/App.cpp

```diff
@@ -2851,8 +2851,6 @@ bool OrbitApp::IsLoadingCapture() const {
 }
 
 ScopedStatus OrbitApp::CreateScopedStatus(const std::string& initial_message) {
-  ORBIT_CHECK(std::this_thread::get_id() == main_thread_id_);
```

Comment:
Not sure I understand the reason to remove the first check - can this actually happen? Or is it now deemed ok to use this from a different thread since we're no longer keeping a raw pointer?

Reply:
Yes, I believed this was thread safe. But I'm not sure anymore. Readding this check. The other one though can go away.